pyodide_build/xbuildenv.py

```diff
@@ -250,7 +250,7 @@ def _download(self, url: str, path: Path) -> None:
             Path to extract the cross-build environment to.
             If the path already exists, raise an error.
         """
-        logger.info("Downloading Pyodide cross-build environment from %s", url)
+        logger.info(f"Downloading Pyodide cross-build environment from {url}")
```

Comment:
I recently learned that it is not recommended not to use f-strings in log calls (https://github.com/pyodide/micropip/pull/131).

Reply:
Thanks for the pointer! I'll keep that in mind and add a Ruff rule for this in another PR.

Reply:
Addressed in de4e3d70c5797ad010e72b9b694a20084ccdccfd.